plugins/inputs/win_wmi/win_wmi_test.go

```diff
@@ -0,0 +1,68 @@
+//go:build windows
+// +build windows
+
+package win_wmi
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+// initialize test data
+var sysDrive = fmt.Sprintf(`%s\`, os.Getenv("SystemDrive")) // C:\
+var logger = new(testutil.Logger)
+var acc = new(testutil.Accumulator)
```

Comment:
Can you please create those in _every_ test separately! The reason is that otherwise, you enter the field of stateful testing (remember both the logger and accumulator are potentially written to) where testing order might matter, new tests might break things and other side-effects might occur!